tuf/client_rework/download.py

```diff
@@ -99,7 +99,8 @@ def download_file(url, required_length, fetcher, STRICT_REQUIRED_LENGTH=True):
     # encoded as %5c in the url, which should also be replaced with a forward
     # slash.
     url = six.moves.urllib.parse.unquote(url).replace("\\", "/")
-    logger.info("Downloading: " + repr(url))
+    msg = f"Downloading: {url}"
+    logger.info(msg)
```

Comment:
I disagree with the whole logging-not-lazy commit: Working around warning like this is not useful.
* if we want to use f-strings with logging, let's silence the warning in the linter config
* alternatively, let's just use %-style strings (I lean this way)